command/server/test-fixtures/config_raft.hcl

```diff
@@ -9,7 +9,7 @@ listener "tcp" {
 }
 
 storage "raft" {
-  path = "./server/test-fixtures/raft_storage_file.db"
+  path = "./server/test-fixtures/"
```

Comment:
could you explain or link a reason for this config change?

Reply:
Absolutely! the boltDB file is created by the raft backend -- the config is meant to have the path to the directory where vault can create this file.